google/cloud/storage/emulator/tests/test_utils.py

```diff
@@ -472,6 +474,151 @@ def test_error_propagation(self):
         self.assertIn("Traceback (most recent call last):", body)
         self.assertIn("TypeError", body)
         self.assertIn("custom message", body)
+        conn.close()
+
+
+class TestRetryTest(unittest.TestCase):
+    def setUp(self):
+        self.emulator = os.getenv("CLOUD_STORAGE_EMULATOR_ENDPOINT")
+        if self.emulator is None:
+            self.skipTest("CLOUD_STORAGE_EMULATOR_ENDPOINT is not set")
+        self.conn = http.client.HTTPConnection(
+            self.emulator[len("http://") :], timeout=10
+        )
+
+    def tearDown(self):
+        self.conn.close()
+
+    def test_retry_test_resource_crud(self):
```

Comment:
We seem to be missing a test to verify that *all* APIs are wrapped. In case I forgot to add `@retry_test(method="fancy.new.method")` when we add `fancy.new.method` to the APIs.

Do you think it is possible to list all the "methods" in the JSON / XML APIs and write a test to verify each of them got a wrapper, maybe using some `flask` introspection or using the discovery doc?  If it is a lot of work, please open a bug as a future enhancement.

Reply:
Filed: https://github.com/googleapis/google-cloud-cpp/issues/6625